docs/dbus/pool.xml

```diff
@@ -745,6 +745,14 @@
       <arg name="return_code" type="q" direction="out" />
       <arg name="return_string" type="s" direction="out" />
     </method>
+    <!-- Extend the data size of a pool. -->
```

Comment:
I think this should be "Extend the available size of a pool" because this space could be used for thinpool metadata too, which might be confusing.

Reply:
Done!